spring-pulsar-cache-provider-caffeine/spring-pulsar-cache-provider-caffeine.gradle

```diff
@@ -23,10 +23,9 @@ shadowJar {
 	}
 	relocate 'com.github.benmanes.caffeine', 'org.springframework.pulsar.shade.com.github.benmanes.caffeine'
 	relocate 'com.google', 'org.springframework.pulsar.shade.com.google'
-	relocate 'org.checkerframework', 'org.springframework.pulsar.shade.org.checkerframework'
```

Comment:
Eventhough main is on caffeine 3.2.1 which no longer includes `org.checkerframework` this is just dead code that needs to be removed.